paasta_tools/monitoring_tools.py

```diff
@@ -70,10 +70,6 @@ def get_page(overrides, service, soa_dir=DEFAULT_SOA_DIR):
     return __get_monitoring_config_value('page', overrides, service, soa_dir)
 
 
-def get_alert_after(overrides, service, soa_dir=DEFAULT_SOA_DIR):
```

Comment:
Gosh, this is kinda embarrassing!

I guess it is better to just be "honest" about what we support, and apparently don't support alert_after or realert_every somehow?

Can you remove these from the docs, or I can in a second pass

Reply:
heh then I have to wait for CI, I can do that though :)

Reply:
actually, I'd be more comfortable letting a second pass pick this up -- as I don't know what internal docs reference this

Reply:
I think the docs are correct on the support actually, since `alert_after` does appear to be configurable, but just doesn't use this function to do overrides: https://github.com/Yelp/paasta/blob/4fba3ff97da4163db622d4f869a52fb1f48f3ca8/paasta_tools/monitoring_tools.py#L250-L251

Similarly, `realert_every` is set to a default but appears configurable? https://github.com/Yelp/paasta/blob/4fba3ff97da4163db622d4f869a52fb1f48f3ca8/paasta_tools/monitoring_tools.py#L206

`check_every` is not though, and makes it clear in the docs: https://github.com/Yelp/paasta/blob/4fba3ff97da4163db622d4f869a52fb1f48f3ca8/docs/source/yelpsoa_configs.rst#L907-L909

It's also unconditionally overridden at https://github.com/Yelp/paasta/blob/4fba3ff97da4163db622d4f869a52fb1f48f3ca8/paasta_tools/monitoring_tools.py#L252

Does this look right, or are these not actually available as options to services? As far as I can tell it is pulling in the monitoring options from services correctly since `monitoring_overrides` ultimately comes from https://github.com/Yelp/paasta/blob/4fba3ff97da4163db622d4f869a52fb1f48f3ca8/paasta_tools/utils.py#L533-L535